src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const additionalInformationDescription = (
+  <div>
+    <p>
+      If there is any other information you would like to provide as part of
+      your claim, please add it here.
+    </p>
+    <p>
+      It‘s important to tell us why your service-connected disability is the
+      reason you‘re no longer able to work. Providing specific examples will
+      help us understand your claim.
+    </p>
+    <p>
+      If you‘ve left one or more jobs because of your service-connected
+      disabilities, please note that.
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': 'Individual Unemployability',
+  'ui:description': additionalInformationDescription,
+  remarks: {
+    'ui:title': ' ',
+    'ui:widget': 'textarea',
+    'ui:options': {
+      rows: 5,
+      maxLength: 3200,
```

Comment:
These are listed as requirements?

Reply:
I just used examples of other textareas and these were the options used.